data/meeting_minutes.yml

```diff
@@ -1,4 +1,5 @@
 years:
```

Comment:
You'll want to remove the top-level `years` field when you are rebasing your changes for PR https://github.com/jakartaee/jakarta.ee/pull/1631.

Reply:
Perfect. Thanks!